crates/egui/src/painter.rs

```diff
@@ -469,6 +469,29 @@ impl Painter {
         }
     }
 
+    /// Paint text that has already been laid out in a [`Galley`].
+    ///
+    /// You can create the [`Galley`] with [`Self::layout`].
+    ///
+    /// The entire [`Galley`] will be rendered with the given opacity in gamma space.
+    #[inline]
+    pub fn galley_with_opacity_factor(
```

Comment:
I think these helpers are too much. `painter.add(TextShape::new(pos, galley, color).with_opacity_factor(opacity))` is ergonomic enough for this rare use case imho

Reply:
Applied in latest commit